pkg/lib/operation/project/remote/table/preview/render.go

```diff
@@ -127,5 +146,27 @@ func measureColumns(table *keboola.TablePreview) []int {
 			}
 		}
 	}
+
+	// then we attempt to fit all of that content on the user's screen
+	// by truncating content as necessary to fit `maxWidth`
+	if adaptWidth {
```

Comment:
V každé iteraci to assigne kus remaining width tomu column, další columns pak můžou mít z toho zbývajícího max `remaining / number of remaining columns`. V praxi teda nebude žádný column příliš velký a většina malých columns nebude truncated.